build-system/pr-check.js

```diff
@@ -517,8 +520,8 @@ function main() {
       colors.cyan(process.env.BUILD_SHARD),
       '\n');
 
-  // If $TRAVIS_PULL_REQUEST_SHA is empty then it is a push build and not a PR.
-  if (!process.env.TRAVIS_PULL_REQUEST_SHA) {
+  const isPushBuild = !process.env.TRAVIS_PULL_REQUEST_SHA;
+  if (isPushBuild) {
```

Comment:
This change is unnecessary and can be reverted. Better still, change it to this...
```
if (process.env.TRAVIS_EVENT_TYPE === 'push')
```
... instead of inferring if it's a push build by looking for the absence of `TRAVIS_PULL_REQUEST_SHA`.

Reply:
Self-documenting vars > comments. 😋 

TRAVIS_EVENT_TYPE is best though, thanks.